sass-style-guide.md

```diff
@@ -59,87 +42,111 @@ You should order your rules from generic to specific. Example:
-
-1. Display:
-  * visibility
-  * opacity
-  * display
-2. Layout:
-  * flex, flex-direction, flex-wrap, align-items, justify-content …
-  * vertical-align
-  * position
-  * z-index
-  * top, right, bottom, left
-  * float
-  * clear
-  * overflow
-3. Box-Model (box-sizing: border-box):
-  * width
-  * height
-  * margin
-  * border
-  * border-radius
-  * padding
-  * outline
-  * transform
-  * box-shadow
-4. Typography:
-  * font
-  * line-height
-  * text-align
-  * text-decoration
-  * white-space
-5. Colors:
-  * color
-  * fill
-  * background
-6. Miscellaneous:
-  * cursor
-  * transition
-  * pointer-events
-  * …
-
-## Syntax conventions
-
-* We use the old **Sass indented** syntax because it is the more rigid syntax of **Sass** and facilitates code homogeneity. The best way to end the fight between different parenthesis placement practictioners is to simply remove parenthesis. ;)
-* Be consistant with indentation – we’re using two spaces.
-* Use english for naming things. The English language has proven itself among coders as the standard.
-* Use lowercase for class names.
-* Multi-word selectors are **separated by hyphens (`-`)**. This is consistent with general CSS conventions including CSS attribute names and Sass functions.
-* Please place **empty lines only between blocks of code that are not nested**. If you are working on a part of the code you already know your way around. But it is important that you get a quick overview if the whole file is not familiar to you. Besides, it’s easier to recognize the nesting structure if nestings are in one block with the parent selector.
-* Be consistent in declaration order, cluster related properties (see above).
-* Be consistant with quotes – we’re using single quotes.
-* Quote attribute values in selectors, e.g. `input[type='checkbox']`.
-* Write one selector per line, one rule per line.
-* Put a space after `:` in property declarations, e.g. `margin: 0`
-* Include a space after each comma in comma-separated property or function values, e.g. `rgba(255, 255, 255, 0)`.
-* Use rgb and rgba color codes, because most of us can envision what color results by mixing red, green and blue.
-
-It could be a quick-win to use [Sass Lint](https://github.com/sasstools/sass-lint) in your projects.
-
-## Coding conventions
-
-### Avoid dangerous selectors
-
-If a selector is too generic, it’s dangerous. In 99% of cases you have to overwrite this rule somewhere. Be more specific. Try using a class instead. (Exception: CSS-Resetstyles)
-
-**bad**
+## Syntax und Formatierung
+
+Eine der einfachsten Formen eines Styleguides ist ein Set von Regeln bezüglich Syntax und Formatierung zu definieren, so dass alle Entwickler eine gemeinsame Basis haben und der Code gut zu lesen ist.
+
+### Schreibweise
+
+- Einrückung: zwei Leerzeichen, keine Tabs
+- Multi-Line-Stil: Ein Selektor pro Zeile, eine Deklaration pro Zeile
```

Comment:
@ChristianPeters Nein, ich habe mich nicht dagegen entschieden. Ich wollte nur erst einmal alles Relevante in die Datei packen. Fürs Styling war gestern keine Zeit mehr. Ich schau heute, was sich da rausholen lässt.